core/src/main/java/org/apache/james/mime4j/stream/MimeEntity.java

```diff
@@ -395,10 +398,13 @@ private void advanceToBoundary() throws IOException {
         }
     }
 
+    @SuppressWarnings( "resource" )
```

Comment:
Don’t suppress warnings unless you know it’s the right thing to do and validate it. Right now it looks like it’s hiding the problem